### DISCUSSION TITLE
2022-04-06 Weekly Meeting 50 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#208] [Weekly Meetings]
Weekly Meeting 50 is happening on Wednesday April 6th, 2022 from 9:00 AM PST(16:00 UTC) - 10:00 AM PST (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

Working Group Updates:

- OMI Website update
- OMI-Chan group
- Scripting group
- [gLTF group](https://github.com/omigroup/gltf-extensions/discussions/76)
- any others?

Workshop Updates:

- Workshop recap
  - Miro Board: https://miro.com/app/board/uXjVOFdPAIQ=/?invite_link_id=889027571587 
- OMI Next Steps

**Comment by mrmetaverse** [comment]
- gltf updates 
- revisit last weeks workshop, add any new thoughts 
- VOTE: proposed minimum requirements for working groups: https://github.com/omigroup/omigroup/issues/176, related to https://github.com/omigroup/omigroup/issues/207
- https://github.com/omigroup/website/issues/39#issuecomment-1089518896
- https://github.com/omigroup/omigroup/issues/202#issuecomment-1086878913
- https://github.com/omigroup/omigroup/issues/195?notification_referrer_id=NT_kwDOA8fQorMzMjQyNjM4MTUzOjYzNDI2NzIy#issuecomment-1069386501
- Proposed Social media and brand strategy group (part of or replacing Comms): https://github.com/omigroup/omigroup/issues/205 forming a proper comms group to manage communications and content planning. This group would need a repo, readme, and backlog. If interested we need to submit an issue proposing the group. At the moment comms and finance and other key pieces of OMI operations are all lumped under Interoperators which may be a risk. 
- Portals group: https://github.com/omigroup/omigroup/issues/209 I will be submitting a proposal to form an exploratory group for portals in the metaverse, this received numerous interested parties during our workshop, and feels in-line with OMI efforts and values. 
- What would blogging look like? https://github.com/omigroup/website/discussions/53
- DO WE WANT TO ADD TWO CHAIRS TO START? We think we can make immediate impact by adding two more chairs, while we continue to evolve as a group. Ideally, these two would fill new roles and unlock new potential for our org, as well as, fill some of the existing gaps and overages. For clarity: This would yield 5 total chairs (which may change again as we go) We have been tracking the future structure of OMI with this milestone https://github.com/omigroup/omigroup/milestone/2

**Comment by mrmetaverse** [comment]
Created blogging milestone in website group @antpb 
- website group milestones: https://github.com/omigroup/website/milestones?with_issues=no 
- first blog issue: https://github.com/omigroup/website/issues/54

**Comment by robertlong** [comment]
![Screen Shot 2022-04-06 at 10 00 04 AM](https://user-images.githubusercontent.com/1753624/162028309-98a31b9d-9aa7-42aa-8940-9a83f9dc730f.png)

![Screen Shot 2022-04-06 at 10 00 38 AM](https://user-images.githubusercontent.com/1753624/162028411-904b85ba-be88-4bf2-838c-0c82fa603d8a.png)
